engine/src/snapshot/migration/v3.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::*;
+
+/// Key size for the ephemeral key
+const KEY_SIZE: usize = 32;
+/// Key type alias.
+pub type Key = [u8; KEY_SIZE];
+
+const VERSION_V3: [u8; 2] = [0x3, 0x0];
+
+#[allow(dead_code)]
+pub(crate) fn read<I: Read>(input: &mut I, password: &Key, _associated_data: &[u8]) -> Result<Vec<u8>, Error> {
```

Comment:
` _associated_data` has been removed in 2.0, do we want to keep it here ?

Reply:
Sure. Removed.